invenio_app_ils/records/api.py

```diff
@@ -155,6 +157,18 @@ def create(cls, data, id_=None, **kwargs):
         """Create Location record."""
         return super(Location, cls).create(data, id_=id_, **kwargs)
 
+    def delete(self, **kwargs):
+        """Delete Location record."""
+        iloc_search = InternalLocationSearch()
+        count = iloc_search.search_by_location_pid(
+            location_pid=self['location_pid']
+        ).count()
+        if count:
+            raise RecordHasReferencesError(
```

Comment:
Does it make sense to also say for which other records it has references?

Reply:
@ntarocco do you mean to display the references also apart from the record that they refer to?

Reply:
Yes maybe to be added later on...

Reply:
Its added now, the error displays also the ids of the reference that cause the problem.